### PR TITLE
changed test_flavor to s3.medium.2

### PR DESCRIPTION
**Pull request by kucerakk**
changed the default flavor to s3.medium.2, but extended with further logic to keep s2.medium.2 in case s3 is not available yet

### CODE REVIEW

File: playbooks/scenario2_simple_ecs.yaml [hunk 1]
```diff
@@ -14,7 +14,17 @@
         test_keypair_name: "{{ (prefix + '-kp') }}"
         test_network_name: "{{ ( prefix + '-net') }}"
         test_image: Standard_Fedora_33_latest
+        test_flavor: s3.medium.2
+
+    - name: Check if s3.medium.2 flavor exists
```
Comment (vladimirhasko) [inline playbooks/scenario2_simple_ecs.yaml:19]: please rename this task name to Query specific flavor
or something similar

File: playbooks/scenario2a_ecs_az.yaml [hunk 2]
```diff
@@ -19,6 +19,16 @@
         test_keypair_name: "{{ (prefix + '-kp') }}"
         test_network_name: "{{ ( prefix + '-net') }}"
 
+    - name: Check if s3.medium.2 flavor exists
```
Comment (vladimirhasko) [inline playbooks/scenario2a_ecs_az.yaml:22]: same here

File: playbooks/scenario2c_coreos_az.yaml [hunk 3]
```diff
@@ -20,6 +20,16 @@
         test_keypair_name: "{{ (prefix + '-kp') }}"
         test_network_name: "{{ ( prefix + '-net') }}"
 
+    - name: Check if s3.medium.2 flavor exists
```
Comment (vladimirhasko) [inline playbooks/scenario2c_coreos_az.yaml:23]: and here

File: playbooks/scenario2_simple_ecs.yaml (hunk 1 above)
Comment (kucerakk) [reply playbooks/scenario2_simple_ecs.yaml]: changed

**Review by gtema — APPROVED** [review]

**Pull request merged**